haystack/backends/__init__.py

```diff
@@ -901,6 +902,10 @@ def add_distance(self, field, point):
             'point': ensure_point(point),
         }
 
+    def _get_facet_fieldname(self, field):
```

Comment:
Do you want to update the other methods that are duplicating this code to use this helper instead? All of the `add_foo_facet` methods?